SQLQuery_3(homework).sql

```diff
@@ -0,0 +1,303 @@
+CREATE DATABASE ZooManagement;
+
+USE ZooManagement;
+
+CREATE TABLE ZooInstitutions (
+    Id INT IDENTITY(1,1) PRIMARY KEY,
+    Name VARCHAR(100) NOT NULL, 
+    City VARCHAR(150) DEFAULT 'Country Romania',
```

Comment:
City could have been added in a separate table where details about addresses are stored.